public/index.js

```diff
@@ -33,6 +34,8 @@ buttons.forEach((button) => {
   button.addEventListener("click", () => {
     console.log(lactoseIntolerant.checked);
     dishOriginCountry = button.value;
+    loadingContainer.style.display = "block";
```

Comment:
@josueJURE this sets the loadingIndcator style to block, making it appear once button is clicked.